src/components/BackOffice/CloseWorkOrders/index.js

```diff
@@ -132,36 +173,39 @@ const CloseWorkOrders = () => {
       })
       .finally(() => {
         setLoading(false)
+        setShowDialog(false)
       })
   }
 
   return (
-    <Layout title="Bulk-close workOrders">
+    <Layout title="Bulk-close Work Orders">
       {loading ? (
         <Spinner />
       ) : (
         <>
           {formSuccess ? (
             <div>
-              <SuccessMessage title="WorkOrders cancelled" />
+              <SuccessMessage title="Work Orders cancelled" />
               <p>
                 <a
                   data-test="closeMoreButton"
                   className="lbh-link"
                   role="button"
                   onClick={() => setFormSuccess(null)}
                 >
-                  Bulk-close more workOrders
+                  Bulk-close more Work Orders
                 </a>
               </p>
             </div>
           ) : (
-            <form onSubmit={handleSubmit}>
+            <form onSubmit={validateForm}>
```

Comment:
I don't think validateForm entirely explains what this method does. HandleSubmit is much better

Reply:
I needed to link the "Yes button" of the modal to an action, so I've put validateForm first, to validate the form for errors, and if there are no errors, then the modal is shown. At this point you know the form has been populated correctly and there are no errors. If then the user presses "Yes/Confirm" the network request is fired. I think it's ok as is but if you've got any suggestions on how to name this method let me know your suggestions